PetAdoptions/petsite/petsite/Controllers/PetHistoryController.cs

```diff
@@ -48,10 +62,28 @@ public async Task<IActionResult> Index()
     [HttpDelete]
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

DeletePetAdoptionsHistory is a state-changing MVC method that does not validate the antiforgery token or do strict content-type checking. State-changing controller methods should either enforce antiforgery tokens or do strict content-type checking to prevent simple HTTP request types from bypassing CORS preflight controls. Learn more - https://cwe.mitre.org/data/definitions/352.html